lib/livebook_web/live/user_component.ex

```diff
@@ -25,11 +29,11 @@ defmodule LivebookWeb.UserComponent do
         User profile
       </h3>
       <div class="flex justify-center">
-        <.user_avatar user={@preview_user} class="h-20 w-20" text_class="text-3xl" />
+        <.user_avatar user={@user} class="h-20 w-20" text_class="text-3xl" />
```

Comment:
Does it show the new color as it's being changed?

Reply:
@aleDsz we should probably revert those changes to the user_avatar/user_modal because iirc they are not using a component anyway? They use hooks which run in the LiveView so flash should always just work.

Reply:
I can take a look if it works without `push_redirect/2`

Reply:
Hmm, it doesn't. And we have the modal as LiveComponent, so it needs to push_redirect/push_patch to something

Reply:
I would say it's good to have a success feedback because we close the modal and it could sound like it didn't nothing

Reply:
My point is that the avatar change is a feedback :D Maybe it's too subtle though, @josevalim do you have an opinion?